tests/testthat/test-bootstrap.r

```diff
@@ -249,7 +249,7 @@ test_that("selectInput selects items by default", {
 
   # Nothing selected when choices=NULL
   expect_true(grepl(fixed = TRUE,
-    '<select id="x" class="form-control"></select>',
+    '<select class="shiny-input-select form-control" id="x"></select>',
```

Comment:
Just FYI, unfortunately, this is likely going cause a bunch of CRAN packages to be updated.

Reply:
I ran revdepchecks for shiny with the updated classes and 4 failures were found across 1136 packages. The failures were **all** (surprisingly!) around snapshots of `actionButton()` markup, so I rolled back the class changes so that we're only adding new classes to inputs that don't currently have a shiny-added class (i.e. the previous generic inputs in the post above).